### DISCUSSION TITLE
【求助】我不小心掉进空洞里了

### DISCUSSION THREAD

**Discussion by bxzwq** [share121/inter-knot#5711] [公告]
我没有掉进空洞的前的记忆，我不知道我是怎么掉进空洞的，空洞信号不太好，拜托，这次一定要成功……

**Comment by bxzwq** [comment]
好像……成功了，空洞里的信号是真的差……

**Reply by Robin-txdy** [reply]
人没事就好

**Comment by KELERSAMA** [comment]
0号空洞吗？如果是就只有法厄同可以救你了 可惜法厄同有一阵没出面接委托了

**Comment by Rae-CY** [comment]
要不试试找绳匠？

**Comment by bxzwq** [comment]
是以骸……

**Comment by Hoyoer** [comment]
七天了，还活着吗？

**Comment by Hoyoer** [comment]
![1731993689031e645513947ad03e61dbbbd48a9b9a0b64080dce5094e2eb9f9c752d14a9c5a11.0.jpg](https://github.com/user-attachments/assets/83f0da84-7e8e-47fe-9a39-8a85b396b161)

**Comment by bxzwq** [comment]
出来了，前几天刚被救出来，我还在空洞里找到一本笔记本，不过空洞里怎么会有这种东西？手机要没电了，我得

**Reply by woaixier** [reply]
能出来就好
